

# Welcome

Our lab is interested in alternative RNA structures and the myriad of roles they have in both viral and human biology. In particular, we study how RNA folding informs alternative splicing and how mis-folding can lead to disease. Our goal is to understand gene expression regulation in physiology and disease through the lens of RNA. 

## Repos

### Data processing and visualization

[SEISMIC-RNA](https://github.com/rouskinlab/SEISMIC-RNA):
SEISMIC-RNA is a tool for DMS-MaPseq processing (dimethyl sulfate mutational profiling with sequencing). It includes DREEM (Detection-of-RNA-folding-Ensembles-using-Expectation-Maximization), an algorithm to distinguish multiple RNA conformations formed by the same underlying sequence in vivo at single nucleotide resolution at a high-throughput scale. 

[SEISMOGRAPH](https://github.com/rouskinlab/seismic-graph)
A tool for SEISMIC-RNA visualization

### eFold

[eFold](https://github.com/rouskinlab/efold):
Our RNA structure prediction tool. Pre-print here: [link](https://www.biorxiv.org/content/10.1101/2024.01.24.577093v3.full.pdf+html).

[eFold data](https://github.com/rouskinlab/efold_data):
The whole data processing pipeline from the source to the model. Uses RouskinHF to push and pull to HuggingFace.

[eFold figures](https://github.com/rouskinlab/efold_figures):
The code for the figures from the [eFold paper](https://www.biorxiv.org/content/10.1101/2024.01.24.577093v3.full.pdf+html).

[RouskinHF](https://github.com/rouskinlab/rouskinhf):
This repo is a Python wrapper around HuggingFace API to pull/push data from the RouskinLab HF organization. 

### Utils

[RNAstructure](https://github.com/rouskinlab/RNAstructure)
This repo is a Python wrapper around Prof. Matthew's RNAstructure software. 
